src/taskgraph/run-task/run-task

```diff
@@ -1053,9 +1054,14 @@ def maybe_run_resource_monitoring():
     monitor_process.start()
     return process
 
+def _display_pyhton_version():
+    print("Python version: ")
+    print(platform.python_version())
```

Comment:
Instead of `print`, you should use `print_line(b"setup", b"<msg>")`. This way it will go to the same place as the rest of the logs and look consistent with them :). I'm not really sure why we need to use byte strings, but oh well.. best to stay consistent.